lib/commands/install.js

```diff
@@ -140,6 +141,21 @@ class Install extends ArboristWorkspaceCmd {
       throw this.usageError()
     }
 
+    if (!isGlobalInstall) {
+      const pkgJson = new PackageJson()
+      await pkgJson.load(this.npm.prefix)
+      const jsonContent = pkgJson.content
+      const isJsonEmptyObject = Object.keys(jsonContent).length === 0
+        && jsonContent.constructor === Object
+
+      const emptyJson = new Error('EEMPTYPKGJSON: package.json is empty object')
```

Comment:
why not remove this check, let it generate empty lock file if package.json is empty. Only make sure not to generate lock file if package.json file doesn't exists

Reply:
If you'll read my last reply you'll see that this check in this place is not the right solution to begin with.